books/models.py

```diff
@@ -605,6 +605,12 @@ def get_community_resource_feature_link_url(self):
         related_name='+',
         help_text='Promote image.'
     )
+    translations = StreamField([
```

Comment:
@edwoodward what is this for?

Reply:
Roy needs to know if a book has translations so he can display a link to the translation. There is no cross reference to translations without creating it manually. This will allow a book to have one or more translations. We will need to add the linkage to the English and the Spanish versions of each book.